ols/app/models/models.py

```diff
@@ -61,11 +61,13 @@ class LLMResponse(BaseModel):
         conversation_id: The optional conversation ID (UUID).
         response: The optional response.
         referenced_documents: The optional URLs for the documents used to generate the response.
+        truncated: Set to True if conversation history was truncated to be within context window.
```

Comment:
@tisnik i would have give this a more explicit name, like `history_truncated` so that it's clearer what it refers to, and so that if we have other "truncation" related metadata we want to include in the response, the field names won't conflcit.  APIs are forever(once we ship), so names matter.